src/views/poi_bloc/Website.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import URI from '@qwant/uri';
+import Telemetry from '../../libs/telemetry';
+
+export default class WebsiteBlock extends React.Component {
+  static propTypes = {
+    block: PropTypes.object,
+    poi: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.clickWebsite = this.clickWebsite.bind(this);
+  }
+
+  clickWebsite() {
+    Telemetry.add('website', 'poi', this.props.poi.meta.source,
+      Telemetry.buildInteractionData(
+        {
+          'id': this.props.poi.id,
+          'source': this.props.poi.meta.source,
+          'template': 'single',
+          'zone': 'detail',
+          'element': 'website',
+        }
+      )
+    );
+  }
+
+  render() {
+    return <div className="poi_panel__info__section poi_panel__info__section--website">
+      <div className="poi_panel__info__section__description">
+        <div className="icon-icon_globe poi_panel__block__symbol" />
+        <div className="poi_panel__block__content">
+          <a className="poi_panel__info__link"
+            href={URI.externalise(this.props.block.url)}
+            no-follow={''}
```

Comment:
I know it's a direct transcription of the existing code, but it's an error. "nofollow" should be another value of the `rel` attribute.

Reply:
:+1: